app/views/posts/index.html.erb

```diff
@@ -10,28 +10,27 @@
   </div>
 
 <% @user.posts.each do |post| %>
-  <div>
+
     <div class="post-data">
       <span><b><%= post.title.upcase %></b></span>
-      <p><%= truncate(post.text, length: 190, separator: '...') %></p>
+      <p><%= truncate(post.text, length: 250, separator: '...') %></p>
       <div class="right-align">
         Comments: <%= post.comments_counter %>
         Likes: <%= post.likes_counter %>
       </div>
     </div>
 
-  <% comments = Comment.where(post_id: post.id) %>
-  <% if comments.length > 0 %>
-    <div class="comment-list">
+    <% comments = Comment.where(post_id: post.id).order(created_at: :desc).limit(5) %>
+    <% if (comments.length > 0) %>
+    <div>
       Comments
       <% comments.each do |comment| %>
       <span><% user = User.find_by(id: comment.author_id).name %></span>
```

Comment:
- [ ] Good to have your project running perfectly now. But please make sure a user can comment and like posts. :pray: Currently, you have now button for liking and commenting on a post as you can see below :point_down: 
  
  ![image](https://user-images.githubusercontent.com/92303891/230677540-22847ed9-5895-40eb-82ad-17baa09dd48a.png)

Reply:
***Hello Reviewer!*** 
Please select by clicking on the post you want to interact with, it will redirect you to that specific post and you will be able to comment and like.

Reply:
This is working perfectly